library/functions.po

```diff
@@ -636,86 +613,80 @@ msgid ""
 msgstr ""
 
 #: ../../library/functions.rst:255
-#, fuzzy
 msgid ""
 "Future statements are specified by bits which can be bitwise ORed together "
 "to specify multiple statements.  The bitfield required to specify a given "
 "feature can be found as the :attr:`~__future__._Feature.compiler_flag` "
 "attribute on the :class:`~__future__._Feature` instance in the :mod:"
 "`__future__` module."
 msgstr ""
-"Future 語句使用位元位來指定，多個語句可以通過按位或來指定。具體特性的位元位可"
-"以通過 :mod:`__future__`  模組中的 :class:`~__future__._Feature` 類的例項的 :"
-"attr:`~__future__._Feature.compiler_flag` 屬性來獲得。"
+"Future 陳述式使用 bits 來指定，可以一起被位元操作 ORed 來指定複數的語句。需要"
```

Comment:
This paragraph is rather difficult to translate (it actually took me about 10 minutes to really understand what this paragraph means). What this paragraph is saying is that the future statements (e.g. `from __future__ import generators`) are transformed into bits that constructs the `flags` argument for `compile` function, each feature in `__future__` has its own representation in bits, e.g. `nested_scopes` is `0b10_000`, `division` is `0b10_000_000_000_000`, and you can use bitwise OR operator to combine these flags, so `nested_scopes.compiler_flag | division.compiler_flag` equals to `0b10_000_000_010_000`, and that is the value you should pass as `flags` when you want to use both `division` feature and `nested_scopes` feature while using `compile` in earlier versions of Python.